src/utils/CMakeLists.txt

```diff
@@ -44,10 +45,17 @@ check_symbol_exists(stat                sys/stat.h   HAVE_STAT)
 check_symbol_exists(exec                unistd.h     HAVE_EXEC)
 check_symbol_exists(P_tmpdir            stdio.h      HAVE_P_TMPDIR)
 
+if(HAVE_WINDOWS_H)
+check_symbol_exists(GetFullPathNameW    windows.h    HAVE_GetFullPathNameW)
+check_symbol_exists(GetFullPathName     windows.h    HAVE_GetFullPathName)
+check_symbol_exists(GetFileAttributesW  windows.h    HAVE_GetFileAttributesW)
+check_symbol_exists(GetFileAttributes   windows.h    HAVE_GetFileAttributes)
+else()
```

Comment:
I guess that line 53-57 can be removed completely since it works on mingw to include `windows.h` as well and POSIX systems neither have these functions or fileapi.h